src/components/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Switch from './Switch';
+import { action } from '@storybook/addon-actions';
+import { useState } from 'react';
+// SECTION 1 Import your component and REACT libary
+
+//SECTION 2: Define default render
+export default {
+  title: 'Switch',
+  component: Switch,
+};
+
+//SECTION 3: Define what is shown on Storybook
+
+export const DefaultSwitch = () => <Switch /> ;
+
+export const DisabledSwitch = () => <Switch disabled/>;
+
+export const DiffSizeSwitch = () => (
+  <div>
+    <h3>sm</h3>
+    <Switch swSize="sm" />
+
+    <h3>md (default)</h3>
+    <Switch swSize="md" />
+  </div>
+);
+
+export const DiffColorSwitch = () => (
```

Comment:
Add more components for color switches and implement pat-ui color convention.

Reply:
I finished